apache/netbeans
# Tomcat module cleanup

Cleanup:
- Remove unnecessary if/return statement
- Add missing braces
- Add missing `@Override` annotations
- Use diamond inference
- Use try-with-resources
- Use StandardCharsets instead of a String
- String concatenation in logger
- Use multi-catch and a more specific catch
- Improve Collection.toArray
- Length one String literal in String.indexOf
- Looks for string concatenation in StringBuilder
- Replace StringBuffer with StringBuilder when not necessary

Testing:
- Full build done
- Verify successful execution of libraries and licenses Ant test
- Verify successful execution of Sigtests
- Verify successful execution of commit-validation
- Verify successful execution of rat-exclusions
- Verify successful execution of `enterprise.tomcat5` unit tests
- Started NetBeans and ensure the log didn't have any ERROR or new WARNINGS

After this, I plan to improve Tomcat/TomEE support for Jakarta EE 9/9.1
